discover/deployment/articles-dxp/03-configuring-liferay/04-securing-liferay/01-saml/02-setting-up-service-provider.markdown

```diff
@@ -0,0 +1,279 @@
+    
+    b.  *Entity ID*: the same name of the IdP. If the IdP is another @product@
+        instance, then it is the same name as the above example. 
+
+    c. *Metadata URL*: The IdP's metadata as a URL or as an XML file.
+
+3. On the *General* tab, the *Enabled* checkbox has been checked.
+
+4. Once *Enabled* checkbox has been checked, the service provider's metadata
+   becomes available:
+
+        [host]:[port]/c/portal/saml/metadata
+
+## Changing the SAML Service Provider Settings [](id=changing-the-saml-service-provider-settings)
+
+If you'd like to configure @product@'s SAML Service Provider Settings, navigate to
+the Service Provider tab of the SAML Admin portlet.
+
+The Service Provider tab includes these options:
+
+**Require Assertion Signature?:** When this box is checked, SAML assertions
+must be individually signed in addition to the entire SAML message.
+
+**Clock Skew:** Clock skew is a tolerance in milliseconds used by the Service
+Provider for verifying expiration of messages and assertions. This can be used
+to mitigate time differences between the clocks of the Identity Provider and
+the Service Provider. This usually only matters when assertions have been made
+to expire very quickly.
+
+**LDAP Import Enabled:** When this box is checked, user information is imported
+from the configured LDAP connection based on the resolved NameID. LDAP
+connections can be configured from Portal Settings.
+
+**Sign Authn Requests:** When this box is checked, the AuthnRequest is signed
+even if the Identity Provider metadata indicates that it's not required.
+
+**Sign Metadata:** When this box is checked, the metadata XML file is signed.
+
+**SSL Required:** When this box is checked, any SAML messages that are not sent
+over HTTPS are rejected. This does not affect how URLs are generated.
+
+The Identity Provider Connection page includes these options:
+
+**Name:** The name of the Identity Provider with which to connect.
+
+**Entity ID:** The Identity Provider's entity ID. This value must match the
+entity ID declared in the Identity Provider metadata.
+
+**Clock Skew:** Clock skew is a tolerance in milliseconds used by the Service
+Provider for verifying expiration of messages and assertions. This can be used
+to mitigate time differences between the clocks of the Identity Provider and
+the Service Provider. This usually only matters when assertions have been made
+to expire very quickly.
+
+**Force Authn:** When this box is checked, the Service Provider asks the
+Identity Provider to re-authenticate the user before verifying the user.
+
+**Metadata:** You can either provide a URL to the Identity Provider metadata
+XML file or you can manually upload it. If you provide a URL, the XML file is
+automatically retrieved and periodically polled for updates. You can change the
+update interval in `portlet.properties` by modifying the
+`saml.metadata.refresh.interval` property which specifies a number of seconds.
```

Comment:
As before: saml.metadata.refresh.interval should be set via System Settings ("SAML Configuration") or via config file named com.liferay.saml.runtime.configuration.SamlConfiguration.cfg

Reply:
I'm mentioning only System Settings for these because in our config file documentation, we tell people it's best to create the files by exporting them out of System Settings. Please let me know if this is not enough.

Reply:
Sounds good. That should be enough then.